compiler+runtime/test/jank/form/case/pass-simple-case-no-default.jank

```diff
@@ -0,0 +1,56 @@
+; does not work in jank yet
+;; test case in value position
```

Comment:
What doesn't work here? Let's have a TODO explaining what's needed to make it work.